api/src/handlers/commissions.rs

```diff
@@ -41,8 +44,22 @@ pub async fn handler(
             let commissions = context.read().await.cache.get_commissions(vote_key);
 
             Ok(match commissions {
-                Some(commissions) => {
-                    warp::reply::with_status(json(&ResponseCommissions { commissions }), StatusCode::OK)
+                Some(mut commissions) => {
+                    commissions = commissions
+                        .iter()
+                        .filter(|v| {
+                            v.epoch_start_at
+                                > query_params
+                                    .query_from_date
+                                    .clone()
+                                    .unwrap_or(Utc::now() - Duration::weeks(5))
```

Comment:
this seems suspicious (as in: why 5 weeks? and if it is intended, this should be a constant anyways)